spec/models/comment_spec.rb

```diff
@@ -1,7 +1,31 @@
 require 'spec_helper'
 
 describe Comment do
-  it { is_expected.to validate_presence_of(:nickname) }
-  it { is_expected.to validate_presence_of(:comment) }
-  it { is_expected.to validate_length_of(:comment) }
+  let (:comment) { described_class.new }
+  describe 'Validations' do
+    context 'comment without nickname' do
+      it 'return false' do
+        comment.comment = 'Foo'
+
+        expect(comment).not_to be_valid
+      end
+    end
+
+    context 'comment without a comment' do
+      it 'return false' do
+        comment.nickname = 'Bar'
+
+        expect(comment).not_to be_valid
+      end
+    end
+
+    context 'comment with comment and nickname' do
+      it 'return true' do
+        comment.comment = 'Foo'
+        comment.nickname = 'Bar'
+
+        expect(comment).to be_valid
+      end
+    end
+  end
```

Comment:
Guess we just need to decide on what the convention will be? Personally i like `shoulda_matchers` think they give a really nice concise syntax